runtime/drivers/catalog.go

```diff
@@ -87,3 +90,10 @@ type FindModelSplitsOptions struct {
 	AfterIndex   int
 	AfterKey     string
 }
+
+// InstanceHealth represents the health of an instance.
+type InstanceHealth struct {
+	InstanceID string    `db:"instance_id"`
+	Health     []byte    `db:"health"`
+	CreatedOn  time.Time `db:"created_on"`
```

Comment:
I would propose this schema instead:
```suggestion
	InstanceID string    `db:"instance_id"`
	HealthJSON []byte    `db:"health_json"`
	UpdatedOn  time.Time `db:"updated_on"`
```

- Having `_json` clarifies the field content type when it's not clear from the SQL type
- The semantics here is `updated_on`, not `created_on` (it gets changed after first insertion)